eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -0,0 +1,127 @@
+# Common Changelog Operations
+
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)(\s(?<releaseStatus>\(Unreleased\)|\(\d{4}-\d{2}-\d{2}\)))?)"
+
+# Returns a Collection of ReleaseNotes object containing changelog info for all version present in the gived CHANGELOG
+function Get-ReleaseNotes {
+    param (
+        [Parameter(Mandatory = $true)]
+        [String]$ChangeLogLocation
+    )
+    
+    $ErrorActionPreference = 'Stop'
+
+    $releaseNotes = @{}
```

Comment:
We should add a check that causes an error if the changeloglocation given doesn't exist.